buildprocess/rspack.make.config.js

```diff
@@ -62,11 +66,25 @@ module.exports = function (devMode) {
       ]
     }
   };
+  if (devMode) {
+    // Enable cache with devMode.
```

Comment:
The cache is new in Rspack 1.2, I don't know if enabling it gives a performance improvement.